packages/astro/package.json

```diff
@@ -127,8 +127,6 @@
     "@babel/traverse": "^7.22.5",
     "@babel/types": "^7.22.5",
     "@types/babel__core": "^7.20.1",
-    "@types/dom-view-transitions": "^1.0.1",
-    "@types/yargs-parser": "^21.0.0",
```

Comment:
I believe this one is not a devdep unless something changed, see: https://github.com/withastro/astro/pull/4683

Reply:
Ah I'll have to check that. I was hoping if it's exposed but not public, the types would be `any` but the user would never see it anyways.

Reply:
I don't see an error when using this branch locally for `astro check`. If I manually set `skipLibCheck: false` (Astro defaults to true), and run with Astro stable latest and this branch, and run `tsc` instead, I get this error in both versions:

<img width="1312" alt="image" src="https://github.com/withastro/astro/assets/34116392/6999be6b-a680-41f1-b162-9d7227cc0398">

Among many other errors (and including Vite). Maybe the `tsc` results are expected. So in the case of `astro check` we're safe here?